main.go

```diff
@@ -441,18 +464,48 @@ func (svr *bridge) handleCall(w http.ResponseWriter, r *http.Request) {
 }
 
 func renderTemplate(templateString string, data interface{}) (string, error) {
-	titleTemplate, err := template.New("tmp").Parse(templateString)
-	if err != nil {
-		return "", fmt.Errorf("error in Template: %s", err)
+	var result string
+	var err error
+	var unsupportedFunc string
+
+	//Excludes unsupported template function calls.
```

Comment:
Since I'm new to these prometheus templating functions, I'm not sure why these aren't supported. Can you add a little bit of color in the comment or README maybe?

Reply:
After a good night's rest and looking at the code again, I was able to make the tweaks to allow all the Grafana Template functions to work with the bridge. I overlooked the externalURL being passed outside the "Alerts" JSON section. There are still a few functions excluded, and these are not part of the Grafana Template Functions, so I kept them out of the bridge options for consistency with Grafana.